### DISCUSSION TITLE
Hit Write Limits because Vercel never charged my card to start billing

### DISCUSSION THREAD

**Discussion by Patel-Jugal** [vercel/community#3311] [Help]
### Summary

I added a card to billing several days ago to start a Pro plan. For some reason it scheduled the plan to start tomorrow, but we hit write limits today and there is no way to start the plan early. It is still asking to add a payment method, but Vercel has it already. We cannot hit our database. This is urgent.

### Steps to Reproduce
![Image 7-19-23 at 9 44 PM](https://github.com/vercel/community/assets/96457534/37f9b7c5-22ab-4bf4-b389-38e6c6d67b78)

![54870fe7-7e60-4048-8240-d04a71590144](https://github.com/vercel/community/assets/96457534/c4a0f193-e32b-4727-bb66-e2db91d5c380)

**Comment by amyegan** [comment]
Sorry you're having trouble with this, @Patel-Jugal. If you share the team slug or team ID, then I can look into this for you.
